src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: calc(7 * ($day-square));
```

Comment:
You need to adjust your width calculations to the readme. It says that the calendar needs padding and gaps (flex gap) between the cells. Please take that into account ;)